enterprise/api/v1/views.py

```diff
@@ -369,6 +369,13 @@ def course_detail(self, request, pk, course_key):  # pylint: disable=invalid-nam
         enterprise_customer_catalog = self.get_object()
         course = enterprise_customer_catalog.get_course(course_key)
         if not course:
+            error_message = _(
+                '[Enterprise API] CourseKey not found in the Catalog. CourseKey: {course_key}, Catalog: {catalog_id}'
+            ).format(
+                course_key=course_key,
```

Comment:
Let's change `CourseKey:`  to `Course: `

Reply:
Done